R/linter_tags.R

```diff
@@ -70,7 +70,7 @@ available_linters <- function(packages = "lintr", tags = NULL, exclude_tags = "d
   if (!file.exists(csv_file)) {
     return(empty_linters())
   }
-  available <- utils::read.csv(csv_file, encoding = "UTF-8", as.is = TRUE)
+  available <- read.csv(csv_file, encoding = "UTF-8", as.is = TRUE)
```

Comment:
AFAICT this is the only usage of `read.csv()`. If that is the case, I wouldn't import it.

Reply:
I went back and forth a few times on this, deciding to just go ahead and import it since it's a default package & "everyone" knows where `read.csv()` comes from, so may as well import it. Ditto for `data()` and `packageVersion()`. I would either import all or none of those 3.

Reply:
Re `data()` see my comment - I wouldn't import but instead get rid of the call if it's possible to reuse `namespace_linter()` logic.

Re `packageVersion()` I don't mind because it's used multiple times.

I found myself having to look up which package these "base" functions come from, so the ns-prefixed version helps when deciding if a base Imports: is necessary. But that's not a big deal I think.

All in all maybe not import from `utils`?